4week/src/components/LpAdd.tsx

```
@@ -0,0 +1,201 @@
+import useImageUpload from "../hooks/mutations/useImageUpload"; 
+import { LoadingSpinner } from "./LoadingSpinner";
+
+interface LpAddProps {
+  isOpen: boolean;
+  onClose: () => void;
+}
+
+const LpAdd = ({ isOpen, onClose }: LpAddProps) => {
+  const [tags, setTags] = useState<string[]>([]);
+  const [inputTag, setInputTag] = useState("");
+  const [selectedFile, setSelectedFile] = useState<File | null>(null);
```

ESLint

[error] 'selectedFile' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)